src/goto-instrument/accelerate/accelerate.cpp

```diff
@@ -651,14 +673,16 @@ int acceleratet::accelerate_loops()
 
 void accelerate_functions(
   goto_modelt &goto_model,
-  bool use_z3)
+  bool use_z3,
+  const double random_seed,
+  const double random_var_freq)
 {
```

Comment:
Threading every option through every function down to where you build the solver does not seem like the right way of doing this.